chandra_aca/star_probs.py

```diff
@@ -213,50 +186,67 @@ def prob_n_acq(star_probs):
     return n_acq_probs, np.cumsum(n_acq_probs)
 
 
-def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False, halfwidth=120):
+def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False, halfwidth=120,
+                     model='default-for-date'):
     """
-    Return probability of acquisition success for given date, temperature and mag.
+    Return probability of acquisition success for given date, temperature, star properties
+    and search box size.
 
     Any of the inputs can be scalars or arrays, with the output being the result of
     the broadcasted dimension of the inputs.
 
-    This is based on the dark model and acquisition success model presented
-    in the State of the ACA 2013, and subsequently updated to use a Probit
-    transform and separately fit B-V=1.5 stars.  This is available in the
-    state_of_aca repo as fit_sota_model_probit.ipynb.
+    The probability ``model`` can be specified as 'sota' or 'spline'.  If not specified
+    then the model is chosen based on the ``date``.  If before 2018-04-28T00:00:00
+    then it uses 'sota', otherwise 'spline'.
 
     :param date: Date(s) (scalar or np.ndarray, default=NOW)
     :param t_ccd: CD temperature(s) (degC, scalar or np.ndarray, default=-19C)
     :param mag: Star magnitude(s) (scalar or np.ndarray, default=10.0)
     :param color: Star color(s) (scalar or np.ndarray, default=0.6)
-    :param spoil: Star spoiled (boolean or np.ndarray, default=False)
+    :param spoiler: Star spoiled (boolean or np.ndarray, default=False)
     :param halfwidth: Search box halfwidth (arcsec, default=120)
+    :param model: probability model ('default-for-date' | 'sota' | 'spline')
 
     :returns: Acquisition success probability(s)
     """
     from .dark_model import get_warm_fracs
```

Comment:
Do you want to make this conditional on needing the sota model?

Reply:
Yes, done.